script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -824,7 +825,7 @@ I couldn't bring myself to finish asking if it hadn't healed yet.
 [sha:e5a9cf299fd7e0cbb2c53ead4ab5a1be5ba87fc8]{
 -- Page 328, Offset 17389.
 -- だって、それは俺を<庇|かば>って出来た傷なんだから。
-I mean, she got this wound protecting me after all.
+I mean, she got that wound protecting me after all.
```

Comment:
I mean, she got that wound protecting me, after all.

Reply:
Not sure about this one. I think it reads better without the pause.